src/protocol.ts

```diff
@@ -34,8 +34,6 @@ export interface Endpoint extends EventSource {
 
 export const enum WireValueType {
   RAW = "RAW",
-  PROXY = "PROXY",
-  THROW = "THROW",
```

Comment:
I've removed those because they are unused. If you'd rather keep them, let me know and I will update my PR.